scripts/document.sh

```diff
@@ -36,33 +43,36 @@ perl -pi -e "s/\\$\\{SHORT_VERSION\\}/${SHORT_VERSION}/" "${README}"
 # http://stackoverflow.com/a/4858011/4585461
 echo "## Changes in version ${RELEASE_VERSION}" >> "${README}"
 sed -n -e '/^## /{' -e ':a' -e 'n' -e '/^## /q' -e 'p' -e 'ba' -e '}' CHANGELOG.md >> "${README}"
-
-# Blow away any includes of MapboxCoreNavigation, because
-# MapboxNavigation-Documentation.podspec gloms the two targets into one.
-# https://github.com/mapbox/mapbox-navigation-ios/issues/2363
-find Sources/Mapbox{Core,}Navigation/ -name '*.swift' -exec \
-    perl -pi -e 's/\bMapboxCoreNavigation\b/MapboxNavigation/' {} \;
-find Sources/Mapbox{Core,}Navigation/ -name '*.[hm]' -exec \
-    perl -pi -e 's/([<"])MapboxCoreNavigation\b/$1MapboxNavigation/' {} \;
+    
+PROJECT="MapboxNavigation-SPM.xcodeproj"
+DESTINATION="generic/platform=iOS"
+"${SOURCEKITTEN_PATH}" doc --module-name MapboxCoreNavigation -- -project "${PROJECT}" -destination "${DESTINATION}" -scheme MapboxCoreNavigation > core.json
+"${SOURCEKITTEN_PATH}" doc --module-name MapboxNavigation -- -project "${PROJECT}" -destination "${DESTINATION}" -scheme MapboxNavigation > ui.json
 
 bundle exec jazzy \
-    --podspec MapboxNavigation-Documentation.podspec \
     --config docs/jazzy.yml \
     --sdk iphonesimulator \
-    --module-version ${SHORT_VERSION} \
     --github-file-prefix "https://github.com/mapbox/mapbox-navigation-ios/tree/${BRANCH}" \
     --readme ${README} \
     --documentation="docs/guides/*.md" \
     --root-url "${BASE_URL}/navigation/api/${RELEASE_VERSION}/" \
     --theme ${THEME} \
     --output ${OUTPUT} \
-    --module_version ${RELEASE_VERSION}
+    --module_version ${RELEASE_VERSION} \
+    --sourcekitten-sourcefile core.json,ui.json \
+    2>&1 | tee docs.output
+    
+if egrep -e "(WARNING)|(USR)" docs.output; then
+    echo "Please eliminate Jazzy warnings"
```

Comment:
Could you add `docs.output` to artifacts or print it here?

Reply:
The output that's captured in this file will also be printed to the console, thanks to `tee`.
This output will also be the last one in the script, so no need to reprint it or save otherwise.